source/features/quick-file-edit.tsx

```diff
@@ -37,7 +39,10 @@ async function linkifyIcon(fileIcon: Element): Promise<void> {
 }
 
 function init(signal: AbortSignal): void {
-	observe('.js-navigation-container :not(a) > .octicon-file', linkifyIcon, {signal});
+	observe([
+		'.react-directory-filename-column svg.color-fg-muted',
+		'.js-navigation-container .octicon-file',
```

Comment:
Can `:not(a)` be dropped or was it necessary to exclude symbolic links?

Reply:
I dropped it because:

- `svg` is a direct descendant of `.react-directory-filename-column` (`X > SVG`), but `:not(a)` expects the `X > * > SVG` structure
- with the new observer we don't really need these deduplication mechanisms

Reply:
This selector is not very specific though and I expect errors to crop up with rare icons (git modules, mirrors, etc) so it can be retouched later.

Reply:
It's fine in the new UI. Still, you also changed the selector in the old UI.

Reply:
Oops